free-courses-hi.md

```diff
@@ -44,3 +45,8 @@
 ### Python
 
 * [Python Tutorial For Beginners | Hindi (With Notes)](https://www.youtube.com/playlist?list=PLu0W_9lII9agICnT8t4iYVSZ3eykIAOME) - Harry
+
+
+### Spring Boot
+
+* [Spring Boot Tutorial | Hindi](https://www.youtube.com/watch?v=tRBVD8UE2cQ) - ProgRank
```

Comment:
We prefer YouTube URLs to be to the `/playlist` endpoint rather than the `/watch` endpoint. Could you update this so users can get a better overview of the entire course?

We prefer to use the title as defined by the resource, please don't create your own title.
```suggestion
* [Spring Boot Tutorial For Beginners (in Hindi)](https://www.youtube.com/playlist?list=PL5mjp3QjkuoLPS-L28yKCKyzCMX8WRVno) - ProgRank
```